intro.Rmd

```diff
@@ -95,7 +95,7 @@ It's common to think about modelling as a tool for hypothesis confirmation, and
 
 ## Prerequisites
 
-We've made a few assumptions about what you already know in order to get the most out of this book. You should be generally numerically literate, and it's helpful if you have some programming experience already. If you've never programmed before, you might find [Hands on Programming with R](http://amzn.com/1449359019) by Garrett to be a useful adjunct to this book.
+We've made a few assumptions about what you already know in order to get the most out of this book. You should be generally numerically literate, and it's helpful if you have some programming experience already. If you've never programmed before, you might find [Hands on Programming with R](http://amzn.com/1449359019) by Garrett Grolemund to be a useful adjunct to this book.
```

Comment:
I disagree. We are writing in a friendly, conversational tone—one that invites the reader to be on a first name basis with the authors.

Reply:
Fine. I will undo this one.